src/modules/Dropdown/Dropdown.js

```diff
@@ -1147,6 +1155,15 @@ export default class Dropdown extends Component {
     })
   }
 
+  renderComponent = (component, props) => {
+    switch (component) {
```

Comment:
Rather than a method for this, we should just invoke `React.createElement()` via JSX syntax.  This will allow the user to pass any class or functional component.  They may pass an entirely different component than `Dropdown.Divider`, or one of these wrapped in a HOC which would fail the switch cases.

```diff
-this.renderComponent(component, { ...componentProps, key: i })
+<component { ...componentProps, key: i } />
```

I think the `renderComponent` method then goes away completely.